### ISSUE TITLE
`invalid-str-returned` not detected as a valid option value

### ISSUE THREAD

**Issue by davfsa** [PyCQA/pylint#3358]
### Steps to reproduce
Code to reproduce: 
```py
class Foo:
    def __str__(self):
        return True # pylint: disable=invalid-str-returned
```
(Ik this is invalid, but in our case is pylint not detecting enums correctly)

### Current behavior
Error is still raised because of a bad value being passed

### Expected behavior
Error is suppressed

### pylint --version output
pylint 2.5.0-dev1
astroid 2.3.3
Python 3.8.0 (default, Oct 28 2019, 16:14:01) 
[GCC 9.2.1 20191008]

**Comment by PCManticore** [comment]
Thanks for the report.

I'm not entirely sure what error you are getting, can you post the exact output you get from `pylint`?

**Comment by davfsa** [comment]
@PCManticore sorry about this, but turns out i was just putting the comment in the wrong position, now everything works fine
